todo_app/data/view_model.py

```diff
@@ -0,0 +1,27 @@
+from todo_app.data.item import Item
+
+
+class ViewModel:
+    def __init__(self, items: list[Item]):
+        self._items = items
+
+    @property
+    def items(self):
+        return self._items
+
+    @property
+    def todo_items(self) -> list[Item]:
+        print(self.items)
+        todo_items = []
+        for item in self.items:
+            if item.status == "To Do":
+                todo_items.append(item)
+        return todo_items
```

Comment:
Not a recommendation since I'm not sure where my personal preferences lie, but worth seeing for comparison the more concise list-comprehension approach to this:
```python
    return [item for item in self.items if item.status == "To Do"]
```